websockets-jsr/pom.xml

```diff
@@ -29,11 +29,10 @@
     </parent>
 
     <groupId>io.undertow</groupId>
-    <artifactId>undertow-websockets-jsr</artifactId>
+    <artifactId>undertow-websockets-jsr-jakarta</artifactId>
```

Comment:
Ditto for the module name change.

Reply:
Fixed. We don't use jakarta suffix anymore.